external/fv3fit/tests/training/test_train_dense_autoencoder.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from fv3fit.keras import TrainingLoopConfig
+from fv3fit.tfdataset import tfdataset_from_batches
+from fv3fit.reservoir.domain import concat_variables_along_feature_dim
+from fv3fit.reservoir.autoencoder import (
+    train_dense_autoencoder,
+    DenseAutoencoderHyperparameters,
+)
+from tests.training.test_train import (
+    get_dataset_default,
+    get_uniform_sample_func,
+)
+import fv3fit
+
+
+def test_train_autoencoder():
```

Comment:
You probably want to mark both of these tests with `@pytest.mark.slow`, as I expect they take more than 1s each.

Reply:
Removed this test module, with the new changes the regular tests will run for this training function.